covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -0,0 +1,102 @@
+
+    app.layout = html.Div(children=content, style={'font-family': 'sans-serif'})
+    return app
+
+
+def _get_header_content(prediction_event: PredictionEvent):
+    # TODO: Rewrite the header to be general for both prediction pages.
+    # TODO: Description in the text of this date is wrong.
+    prediction_date_str = prediction_event.date.strftime("%Y-%m-%d")
+    prediction_link = "https://www.facebook.com/permalink.php?"\
+        "story_fbid=10113020662000793&id=2247644"
+    france_link = "https://www.reuters.com/article/us-health-coronavirus-france-toll/" \
+        "french-coronavirus-cases-jump-above-chinas-after-including-nursing-home-tally-idUSKBN21L3BG"
+    return [
+        html.H1(children='COVID-19 predictions of Boďová and Kollár'),
+        html.P(children=[
+            'On 2020-03-30, mathematicians Katarína Boďová and Richard Kollár ',
+            html.A('made predictions about 7 countries', href=prediction_link),
+            f'. The data available up to that point (until {prediction_date_str}) is in the ',
+            html.Span('green zone', style={'color': 'green'}),
+            f'. Data coming after {prediction_date_str} is in the ',
+            html.Span('blue zone.', style=dict(color='blue')),
+            dcc.Markdown(
+                """
+                The predicted number of active cases <em>N</em>(<em>t</em>) on day <em>t</em> is
+                calculated as follows (constants <em>A</em> and <em>T</em><sub><em>G</em></sub>
+                are country-specific):
+                <em>N</em>(<em>t</em>) = (<em>A</em>/<em>T</em><sub><em>G</em></sub>) ⋅
+                (<em>t</em>/<em>T</em><sub><em>G</em></sub>)<sup>6.23</sup> /
+                e<sup><em>t</em>/<em>T</em><sub><em>G</em></sub></sup>
+                """,
+                dangerously_allow_html=True,
+            ),
+        ]),
+        dcc.Markdown("""
+            ### References
+            * [Polynomial growth in age-dependent branching processes with diverging
+              reproductive number](https://arxiv.org/abs/cond-mat/0505116) by Alexei Vazquez
+            * [Fractal kinetics of COVID-19 pandemic]
+              (https://www.medrxiv.org/content/10.1101/2020.02.16.20023820v2.full.pdf)
+              by Robert Ziff and Anna Ziff
+            * Unpublished manuscript by Katarína Boďová and Richard Kollár
+            """),
+        dcc.Markdown(f"""
+            ### Notes about the graphs
+
+            * Dashed lines are the predictions, solid red lines are the real active
+              cases. Black dotted lines mark the predicted maximums.
+            * We've now added new prediction made on 2020-04-13 by the same authors. There's a
+              second dashed line for Italy, Spain, USA and Germany.
+            * France has been excluded, since they [screwed up daily data reporting]({france_link}).
+            """,
+                     dangerously_allow_html=True)
+    ]
```

Comment:
Aký používaš editor? Lebo ti tuším uchádzajú konce riadkov v súboroch.

Reply:
Sublime. When I add a linter it is going to catch these I hope :)